molecule/default/prepare.yml

```diff
@@ -0,0 +1,58 @@
+---
+- name: Prepare
+  hosts: molecule
+  gather_facts: false
+  become: yes
+  become_method: sudo
+  vars_files:
+    - vars/vars.yml
+
+  tasks:
+    - name: Create a non-root sudoer user
+      ansible.builtin.user:
+        name: "{{ tas_single_node_user }}"
+        shell: /bin/bash
+        create_home: yes
+
+    - name: Grant passwordless sudo to new user
+      ansible.builtin.lineinfile:
+        path: /etc/sudoers
+        line: "{{ tas_single_node_user }} ALL=(ALL) NOPASSWD: ALL"
+        validate: "visudo -cf %s"
+
+    - name: Create an .ssh directory
+      ansible.builtin.file:
+        path: "/home/{{ tas_single_node_user }}/.ssh"
+        state: directory
+        mode: "0700"
+        owner: "{{ tas_single_node_user }}"
+
+    - name: Configure SSH access for new user
+      ansible.builtin.copy:
+        src: "{{ molecule_ephemeral_directory }}/id_rsa.pub"
+        dest: "/home/{{ tas_single_node_user }}/.ssh/authorized_keys"
+        mode: "0600"
+        owner: "{{ tas_single_node_user }}"
+
+    - name: Edit SSH config to disallow root login
+      ansible.builtin.lineinfile:
+        path: /etc/ssh/sshd_config
+        regexp: '^(#*)?PermitRootLogin'
+        line: PermitRootLogin no
+      notify: Restart SSH
+
+    - name: Modify Molecule inventory
+      ansible.builtin.lineinfile:
+        path: "{{ molecule_ephemeral_directory }}/inventory/molecule_inventory.yml"
+        regexp: 'ansible_user:'
+        line: "ansible_user: {{ tas_single_node_user }}}"
```

Comment:
The three `}` chars at the end look like a typo (?)

Reply:
This would be the ending of a json object , so the additional } is to actually close it off